doc/input-output-reference/src/overview/group-surface-construction-elements.tex

```diff
@@ -4120,7 +4120,7 @@ \subsubsection{Inputs}\label{inputs-38}
 
 \paragraph{Field: Tube Spacing}\label{field-tube-spacing}
 
-This field defines the distance between adjacent hydronic tubes spaced in the direction perpendicular to the main direction of heat transfer. The value for this parameter must be greater than 0.01m (or a tube spacing of 1 cm) and less than 1.0m (or a tube spacing of 1m). Note that this parameter is only used for two-dimensional solutions (see previous field) or when the user requests that the tube length for a hydronic radiant system ( \textbf{\hyperref[zonehvaclowtemperatureradiantvariableflow]{variable flow}} or  \textbf{\hyperref[zonehvaclowtemperatureradiantconstantflow]{constant flow}}) be autosized.  In the case of autosizing the tube length, this parameter is used along with the dimensions of the surface to approximate the tube length.
+This field defines the distance between adjacent hydronic tubes spaced in the direction perpendicular to the main direction of heat transfer. The value for this parameter must be greater than or equal to 0.01m (or a tube spacing of 1 cm) and less than or equal to 1.0m (or a tube spacing of 1m). Note that this parameter is only used for two-dimensional solutions (see previous field) or when the user requests that the tube length for a hydronic radiant system ( \textbf{\hyperref[zonehvaclowtemperatureradiantvariableflow]{variable flow}} or  \textbf{\hyperref[zonehvaclowtemperatureradiantconstantflow]{constant flow}}) be autosized.  In the case of autosizing the tube length, this parameter is used along with the dimensions of the surface to approximate the tube length.
```

Comment:
I don't see from the PR description anything about the tube size issue.  Is that just something you identified while fixing the zone index issue and added it here?